src/table.ts

```diff
@@ -59,7 +59,11 @@ export interface TableProps<
     : undefined;
 }
 
-export type AnyTable = ITable<object, keyof object, keyof object | undefined>;
+export type AnyTable = ITable<
+  Record<string | number | symbol, any>,
+  string | number | symbol,
+  string | number | symbol | undefined
```

Comment:
is there a valid case with `number | symbol`?

Reply:
I was trying to `Exclude<keyof object, number | symbol>` instead

Reply:
For `AnyTable`, i think better to leave it open so that `any` will work. 

<img width="321" alt="image" src="https://user-images.githubusercontent.com/38672686/173978197-4d657a22-b4bc-4722-9e87-35726604ea6f.png">

Reply:
Hmm nevermind, doesn't seem to affect anything. Will do this

Reply:
`keyof object` doing weird things for me ...
<img width="306" alt="image" src="https://user-images.githubusercontent.com/38672686/173978378-608ee42f-fafe-4ccb-a17c-9496a2d203ec.png">